drivers/hmis/lib/form_data/default/fragments/hopwa_disability_for_annual.json

```diff
@@ -0,0 +1,24 @@
+{
+  "fragment": "#hopwa_disability",
+  "item": [
+    {
+      "type": "CHOICE",
+      "link_id": "q_4_08_2",
```

Comment:
RE: other comment for removing enable_when's, we can give this a unique link_id for clarity, like `q_4_08_2_hiv_aids_hopwa`. I think we still want this, to ensure that `DisabilityResponse:1` gets set. (The alternative would be to infer it in the processor if any of the hiv aids fields are present).

Reply:
Sounds good and makes sense. After submitting an annual assessment locally in a hopwa project, I checked in the Disabilities table and can see a row where DisabilityType is 8, DisabilityResponse is 1, DataCollectionStage is 5 (annual), and the hopwa questions (TCellCount, TCellSource etc) match what I entered, so I think this is working correctly. 👍